docs/guide/taxonomy/taxonomy.md

```diff
@@ -28,16 +28,30 @@ The identifiers correspond to Field names defined in the Content Type. The `name
 ## Customize taxonomy structure
 
 You can create other taxonomies than the one predefined in the system, for example a Content category.
-To do it, first, create a new Content Type with `content_category` identifier and include the following Field Types:
 
-* `category_identifier` of `ezstring` type
-* `parent` of `ibexa_taxonomy_entry` type
-* `name` of `string` type
+To do it, first, create a new container to store the new taxonomy's items, for example a `folder` named Content categories.
```

Comment:
A nice trick here is to open the camouflaged _Taxonomy_ folder (admin/)view/content/56/full/1/57 and to create this container along side _Tags_.

I don't know if it's a good idea to document this nor how it could be documented.

Reply:
Any advice?